Goal: Adjustments in Postfix templates

Author: Peter Eisentraut <petere@debian.org>

Upstream status: Specific to Debian, parts can be included

Index: b/templates/main.cf.template.in
===================================================================
--- a/templates/main.cf.template.in
+++ b/templates/main.cf.template.in
@@ -22,16 +22,6 @@
 message_size_limit = 20971520
 @@@endif@@@
 
-#   paths
-command_directory = @sbindir@
-daemon_directory = @libexecdir@/postfix
-queue_directory = @emailserver_localstatedir@
-
-#   users
-mail_owner= @emailserver_mail_owner@
-setgid_group= @emailserver_setgid_grp@
-default_privs= @emailserver_default_privs@
-
 #   local host
 myhostname = @@@fqdnhostname@@@
 mydomain = @@@postfix-mydomain@@@
Index: b/templates/master.cf.template.in
===================================================================
--- a/templates/master.cf.template.in
+++ b/templates/master.cf.template.in
@@ -1,6 +1,6 @@
 KOLAB_META_START
 TARGET=@emailserver_confdir@/master.cf
-PERMISSIONS=0640
+PERMISSIONS=0644
 OWNERSHIP=@emailserver_usr@:@emailserver_grp@
 KOLAB_META_END
 # (c) 2004 Steffen Hansen <steffen@klaralvdalens-datakonsult.se> (Klaralvdalens Datakonsult AB)
Index: b/templates/transport.template.in
===================================================================
--- a/templates/transport.template.in
+++ b/templates/transport.template.in
@@ -1,6 +1,6 @@
 KOLAB_META_START
 TARGET=@emailserver_confdir@/transport
-PERMISSIONS=0640
+PERMISSIONS=0644
 OWNERSHIP=@emailserver_tables_usr@:@emailserver_grp@
 RUNONCHANGE=@sbindir@/postmap -o @emailserver_confdir@/transport
 KOLAB_META_END
Index: b/templates/virtual.template.in
===================================================================
--- a/templates/virtual.template.in
+++ b/templates/virtual.template.in
@@ -1,6 +1,6 @@
 KOLAB_META_START
 TARGET=@emailserver_confdir@/virtual
-PERMISSIONS=0640
+PERMISSIONS=0644
 OWNERSHIP=@emailserver_tables_usr@:@emailserver_grp@
 RUNONCHANGE=@sbindir@/postmap -o @emailserver_confdir@/virtual
 KOLAB_META_END
